go/test/endtoend/cluster/vttablet_process.go

```diff
@@ -410,6 +410,13 @@ func (vttablet *VttabletProcess) CreateDB(keyspace string) error {
 	return err
 }
 
+// CreateDBWithSuperReadOnly creates the database for keyspace
+func (vttablet *VttabletProcess) CreateDBWithSuperReadOnly(keyspace string) error {
+	_, _ = vttablet.QueryTabletWithSuperReadOnlyHandling(fmt.Sprintf("drop database IF EXISTS vt_%s", keyspace), keyspace, false)
+	_, err := vttablet.QueryTabletWithSuperReadOnlyHandling(fmt.Sprintf("create database IF NOT EXISTS vt_%s", keyspace), keyspace, false)
+	return err
+}
```

Comment:
I would like to know why we're adding this. Feels like its primary function is to avoid really testing the new work. No?

Reply:
fixed.